### DISCUSSION TITLE
Question abou the scaling vector trained in the EM step

### DISCUSSION THREAD

**Discussion by DongzeHE** [theislab/scvelo#413] [Q&A] [answered]
Recently I have been using your amazing tool, scVelo, a lot to analyze my data. To better understand the results I dug into the code of scVelo. Almost all the steps are clear to me except the **scaling factor "scaling"** used in the EM step. Moreover, it seems like the scaling factor is trained in the EM step, which makes me more confused. 

Could anybody please briefly explain the role of this scaling factor, and what's the conceptual reason for scaling unspliced counts? In addition, why do you need to train this scaling factor in the EM step?

**Comment by VolkerBergen** [comment, marked as answer]
Hi @DongzeHE, 
the docs is currently being updated, so a full description of all parameters is soon to come.

The scaling factors are learned jointly with the other parameters in order to ensure that spliced and unspliced live on the same scale. That is done for two reasons: 
1) The proportions are quite unequal (10-25% for unspliced) and gene-dependent. 
2) The scaling further guarantees that the estimates are scale-invariant. So if you multiplied the counts with some constant (e.g. due to different normalization procedure), you'd still get the exact same rates, latent times etc..
